lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -44,6 +44,9 @@ class SynchronizeLdm < BaseAction
 
         description 'Specifies how to synchronize LDM and resolve possible conflicts'
         param :synchronize_ldm, instance_of(Type::SynchronizeLDM), required: false, default: 'diff_against_master_with_fallback'
+
+        description 'Allows to have include deprecated with model diff/view'
```

Comment:
please update description: Enables handling of deprecated objects in the logical data model.

Reply:
Updated